sdks/java/io/hbase/src/main/java/org/apache/beam/sdk/io/hbase/HBaseMutationCoder.java

```diff
@@ -65,4 +70,44 @@ private static MutationType getType(Mutation mutation) {
       throw new IllegalArgumentException("Only Put and Delete are supported");
     }
   }
+
+  /**
+   * Returns a {@link CoderProvider} which uses the {@link HBaseMutationCoder} for
+   * {@link Mutation messages}.
+   *
+   * <p>This method is invoked reflectively from {@link DefaultCoder}.
```

Comment:
Drop `<p>This method is invoked reflectively from {@link DefaultCoder}.` since I don't expect users to use `@DefaultCoder(HBaseMutationCoder.class)` to set the default coder for a custom user defined type.